requirements/base.txt

```diff
@@ -2,7 +2,6 @@
 elasticsearch==2.1.0
 Django==1.9.8
 djangorestframework==3.4.0
-psycopg2==2.6.2
```

Comment:
`tox` will work, yes. It uses [own requirements](https://github.com/kinaklub/next.filmfest.by/blob/8bfd1f7b28343c6460c0403a14d909fa7f7ebad2/tox.ini#L10) file and I've added `psycopg2` to it below.